src/net.cpp

```diff
@@ -1352,23 +1354,27 @@ void ThreadSocketHandler()
                 {
                     LOG(NET, "Node %s socket no message in first 60 seconds, %d %d from %d\n", pnode->GetLogName(),
                         pnode->nLastRecv != 0, pnode->nLastSend != 0, pnode->id);
-                    pnode->fDisconnect = true;
+                    if (ignoreNetTimeouts.Value() == false)
+                        pnode->fDisconnect = true;
                 }
                 else if (nTime - pnode->nLastSend > TIMEOUT_INTERVAL)
                 {
                     LOG(NET, "Node %s socket sending timeout: %is\n", pnode->GetLogName(), nTime - pnode->nLastSend);
-                    pnode->fDisconnect = true;
+                    if (ignoreNetTimeouts.Value() == false)
+                        pnode->fDisconnect = true;
```

Comment:
I'm wondering if you still want these log messages printed out or not...maybe it doesn't matter, but if so you could do just a single check for ingnoreNetTimeouts at the top of this section and if true then skip the section.

Reply:
I do want the logs in case you are chasing down a problem with them.  The logs don't say "...therefore disconnecting", they just inform that a timeout happened.